plugins/ipam/host-local/host_local_test.go

```diff
@@ -289,4 +323,203 @@ var _ = Describe("host-local Operations", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(strings.Index(string(out), "Error retriving last reserved ip")).To(Equal(-1))
 	})
+
+	It("allocates a custom IP when requested by config args", func() {
```

Comment:
Do we need an "allocates a custom IP..." test case for IPv6, or is the custom IP logic agnostic of family?